openpype/hosts/maya/api/lib.py

```diff
@@ -2818,3 +2818,21 @@ def set_colorspace():
     cmds.colorManagementPrefs(e=True, renderingSpaceName=renderSpace)
     viewTransform = root_dict["viewTransform"]
     cmds.colorManagementPrefs(e=True, viewTransformName=viewTransform)
+
+    # forth set policy path
+    if root_dict.get("policyFilePath"):
+        unresolved_path = root_dict["policyFilePath"]
+        policy_paths = unresolved_path[platform.system().lower()]
+
+        resolved_path = None
+        for policy_p in policy_paths:
+            resolved_path = str(policy_p).format(**os.environ)
```

Comment:
Is it necessary to use multipath? This seems to be serving to the first working path and will of course work, as log as it exists but is it necessary to use multipath if there is only one path which can be used as it is used in line 2833. This might be confusing for users who want to configure it.